minos/networks/messages.py

```diff
@@ -59,6 +74,10 @@ def __init__(self, base: Request, action: Callable[[Any, ...], Any]):
         self.action = action
         self._content = None
 
+    @cached_property
```

Comment:
I recommend to avoid using the `cached_property` in this case because is not a computed value, but an accessing to an inner attribute.